scripts/configure-and-start.sh

```diff
@@ -59,5 +59,10 @@ sed -i "s|collectorApiServerUrl: 'http[^']*'|collectorApiServerUrl: 'http://0.0.
 sed -i "s/serveSubscriptions: Boolean(process.env.WS_SAVE_SUBSCRIPTIONS) || false/serveSubscriptions: Boolean(process.env.WS_SAVE_SUBSCRIPTIONS) || true/" src/config.ts
 npm run compile
 
+cd /home/shardeum/
+
+# Sync litestream
+./run-litestream.sh
```

Comment:
this should be conditional execution. we may have existing db file, we wouldn't want to overwrite that